packages/mongo/mongo.d.ts

```diff
@@ -107,6 +107,11 @@ export namespace Mongo {
       byteSize?: number,
       maxDocuments?: number
     ): Promise<void>;
+
+    /**
+     * @deprecated since 2.8. Check migration guide {@link https://guide.meteor.com/2.8-migration}
+     * @see createIndexAsync
+     */
```

Comment:
I would add that is deprecated in the server only.

In the client those methods will still be available

Reply:
Hmm I thought, those will be deprecated too for the client 😵‍💫 how these functions will work on methods in import folder? This topic should also be covered in the guide.

If we will continue on this pr, some of them are server only methods like createIndex. If you want, I can remove others.